### PR TITLE
Kiel Sprottenflotte changed from Nextbike to Donkey Republic

### PR DESCRIPTION
**Pull request by patzi**
Operator changed in Kiel see sprottenflotte.de or https://www.donkey.bike/de/stadte/kiel/

### CODE REVIEW

File: pybikes/data/gbfs.json [hunk 1]
```diff
@@ -913,6 +913,18 @@
             },
             "feed_url": "https://stables.donkey.bike/api/public/gbfs/donkey_gh/gbfs"
         },
+        {
+            "tag": "donkey-kiel",
```
Comment (eskerda) [inline pybikes/data/gbfs.json:917]: If it's the same system, let's keep the previous tag `sprottenflotte-kiel`

**Review by eskerda — CHANGES_REQUESTED** [review]
Hey @patzi, see comment for a minor change, the rest looks good. Thanks for contributing!

**Review by patzi — COMMENTED** [review]
Fixed this, I thought the tags would be operator specific. Thanks :)

**Comment by eskerda** [comment]
No worries, thanks!

**Pull request merged**